content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,414 @@
+6. The default settings for _filmic_ are adjusted by the software according to the exposure bias stored in the photo metadata. Thus, for digital photographers who are accustomed to exposing to the right of the histogram, forcing a manual exposure compensation on the camera body to protect the highlights from clipping (which is the only good way to expose in digital photography, with all due respect to purists who don't understand the technological differences between photoelectric semiconductor and photo-sensitized silver halide ions) will have their pipeline, relative to the scene, set automatically for them (as a first approximation, because filmmakers still can't guess what the exact mid-grey value is for a given scene).
+
+_Filmic v3_ is still available via the last tab of _filmic rgb_. As well as it is possible to update an image from _filmic v3_ to v4 with the same tab. Here a preview of this new _filmic rgb_ and the selection of the version wanted via science color menu:
+
+![filmic rgb v4]({filename}en/filmicv4.png)
+
+Note: the reconstruction of highlights is done by means of wavelet decomposition by iteration, which induces a certain performance penalty. It is possible to disable the reconstruction completely by setting a very high highlight threshold, as the program bypasses the reconstruction if no pixels are detected beyond this threshold. Nevertheless, given the difficulty of convincingly reconstructing highlights, this is an acceptable compromise.
+
+It is also possible to deactivate, in some cases, the default _higlight reconstruction_ module if _filmic_ reconstruct itself the highlights, to preserve more details. Then think of controlling the desaturation of the highlights in order to absorb the magenta highlights that you could create by deactivating this module.
+
+# Others improvements
+
+This 3.2, like the 3.0, is so rich in new features and improvements. This article doesn't fit to be exhaustive but to present the main changes and allow you to understand them. You will find below, in a jumble, a few other improvements to know.
+
+## Better management of the image processing pipeline
+
+As a reminder, darktable 3.0 has not only introduced a new processing pipeline order but also the possibility to change the order of this pipeline by reordering the modules yourself (very advanced function). In fact, some users got lost and had no way to find the default order of 3.0 or the one of 2.6.x and earlier versions.
+
+So, a new permanent module has been added in _darkroom_ (in bottom of right panel) named _module order_. This module allow to check the pipeline order actually applied to the current image: 3.0, original (default order on versions older than 3.0) and custom (personal order that you define, even unintentionally). This same module also allow you to apply 3.0 preset, or original one, but also to save your own custom preset. Those are the order you can find on new filter collections presented before in this article.
```

Comment:
This module allow -> This module allows
This same module also allow -> This same module also allows

Reply:
At least, I'm quite consistent in that misktake :grin: